File: 0455-assign-cookies/0455-assign-cookies.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */
var findContentChildren = function(g, s) {
    g.sort((a,b) => a-b );
    s.sort((a,b)=> a - b);
    let n = g.length,m = s.length,i=0;
    for(let j = 0;  i<n && j < m ; j++) if(s[j] >= g[i]) i++;
    return i;
};

File: 0455-assign-cookies/0455-assign-cookies.cpp
class Solution {
public:
    int findContentChildren(vector<int>& g, vector<int>& s) {
        sort(g.begin(),g.end());
        sort(s.begin(),s.end());
        
        vector<int>div;
        int i = 0,j = 0;
        int count =0;
        for(; i < g.size() && j < s.size();j++){
            if(s[j]>=g[i]) i++;
        }
        return i;
    }
};